core/src/cs/eng1/piazzapanic/stations/Station.java

```diff
@@ -206,4 +207,12 @@ public StationActionUI.ActionAlignment getActionAlignment() {
     public int getId() {
         return id;
     }
+
+    public List<Subject<Chef>> getChefSubjects() {
+        return chefSubjects;
```

Comment:
Could we test roughly the same thing without exposing chefSubjects? We still have access to nearbyChef, and ultimately need to test behaviour (the abstracted thing does what we expect), not underlying implementation.